modules/currency-l0/src/main/scala/org/tessellation/currency/l0/http/P2PClient.scala

```diff
@@ -37,5 +37,5 @@ sealed abstract class P2PClient[F[_]] private (
   val gossip: GossipClient[F],
   val node: NodeClient[F],
   val stateChannelSnapshotClient: StateChannelSnapshotClient[F],
-  val globalSnapshot: L0GlobalSnapshotClient[F]
+  val l0GlobalSnapshotClient: L0GlobalSnapshotClient[F]
```

Comment:
I think refactoring to `l0GlobalSnapshot` is enough. I think we don't add `Client` suffix in the `Client` class.